packages/bpk-docs/src/pages/ModalsPage/ModalsPage.js

```diff
@@ -146,6 +146,17 @@ const ModalsPage = () => (
     ]}
     components={components}
     readme={modalReadme}
+    usageTable={{
+      dos: [
+        'Use when users need to regain context of where they are in the UI',
+        'Use when users need to focus on a separate task/information in relation to their current goal.',
```

Comment:
I think this would be better with the `/information` part removed, as it reads a bit funny. What do you think?

Reply:
We want to keep this as users won't always have a task to complete